CMakeLists.txt

```diff
@@ -172,14 +177,9 @@ if(TRAVIS_CI_BUILD)
   add_definitions(-Werror)
 endif()
 
-if(CMAKE_COMPILER_IS_GNUCC)
-  check_c_compiler_flag(-Og HAS_OG_FLAG)
-else()
-  set(HAS_OG_FLAG 0)
-endif()
```

Comment:
As he said, it might be a bit conservative, but the benefit is minimal so I'll remove it.